src/Sonarr.Http/Extensions/ReqResExtensions.cs

```diff
@@ -32,9 +32,9 @@ public static object FromJson(this Stream body, Type type)
             return Json.Deserialize(value, type);
         }
 
-        public static JsonResponse<TModel> AsResponse<TModel>(this TModel model, HttpStatusCode statusCode = HttpStatusCode.OK)
+        public static JsonResponse<TModel> AsResponse<TModel>(this TModel model, NancyContext context, HttpStatusCode statusCode = HttpStatusCode.OK)
```

Comment:
I don't think we need this anymore. The other place appears ToErrorModel, and there we can just return the ErrorModel instead of as response.